llpc/docs/DdnGraphicsRegisterHandler.md

```diff
@@ -0,0 +1,265 @@
+# Graphics Register Handler
+
+# Introduction
+This framework is introduced to simpilfy the management of register values (which is fetched from SRD) and improve performance on maintaining expressions for these registers. Without this framework, the implementation may easily lead to hardly understandable build results and introduce redundant operations.
+
+# Background
+
+Take `SQ_IMG_SAMP_WORD2` from `/llpc/imported/chip/gfx9/gfx9_plus_merged_registers.h` for example:
```

Comment:
Nitpick: I'd prefer the path not start with a slash. As it is, it looks very much like a global absolute path.

Reply:
Okay, i'll eliminate the slash.